src/Stratis.Bitcoin.Features.PoA/SlotsManager.cs

```diff
@@ -56,11 +56,13 @@ public uint GetMiningTimestamp(uint currentTime)
             if (!this.federationManager.IsFederationMember)
                 throw new Exception("Not a federation member!");
 
+            List<PubKey> federationMembers = this.federationManager.GetFederationMembers();
```

Comment:
Come to think about it do you apply some sorting or is it a first in sort?

Reply:
I do not apply sorting, the order is defined in network and later is defined by the order in which members are added or removed.

Reply:
How is the order of entry enforced?

Reply:
By using a list of items which preserves order